pylsp/plugins/_resolvers.py

```diff
@@ -82,8 +82,7 @@ def resolve(self, completion):
             return self.callback(completion, sig)
         except Exception as e:  # pylint: disable=broad-except
             log.warning(
-                'Something went wrong when resolving label for {completion}: {e}',
-                completion=completion, e=e
+                log.warning(f'Something went wrong when resolving label for {completion}: {e}')
             )
```

Comment:
is there log inside a log now?

Reply:
What do you mean?

Reply:
Oh god, sorry, my bad. Fixing it now.

Reply:
I was testing this on Spyder and copy/pasted the wrong snippet, sorry.

Reply:
Is this the first use of f-string in this code base? f-strings are only available since Python 3.6. Are there minimal Python requirements for pylsp?

Reply:
Yes, this project only supports Python 3.6+:

https://github.com/python-lsp/python-lsp-server/blob/c35cc13fb8cdc8277402048612ed650b60f508c7/setup.py#L46

Reply:
All good now @rchl?